closed/make/Main.gmk

```diff
@@ -63,7 +63,7 @@ test-image : test-image-openj9
 test-image-openj9 : images
 	@+$(MAKE) $(MAKE_ARGS) -f $(SRC_ROOT)/closed/TestImage.gmk
 ifneq ($(COMPILE_TYPE), cross)
-	$(JRE_IMAGE_DIR)/bin/java -version 2>&1 | $(TEE) $(IMAGES_OUTPUTDIR)/test/openj9/java-version.txt
+	$(JRE_IMAGE_DIR)/bin/java -version >$(IMAGES_OUTPUTDIR)/test/openj9/java-version.txt 2>&1; rc=$$?; cat $(IMAGES_OUTPUTDIR)/test/openj9/java-version.txt; exit "$$rc"
```

Comment:
Perhaps setting `pipefail` would be a better way to address this.

Reply:
Do we know that `pipefail` is always available? I wasn't sure about that, which is why I wrote it this way. If someone can vouch though that we have `pipefail` on all platforms (i.e. we won't be running a plain POSIX `sh`, etc.), then I'd be happy to use that instead.

Reply:
The version is printed to stderr, so redirecting to the file in a second run means redirecting stderr. With stderr redirected, any output indicating a crash in that second run would be hidden from the build log

Reply:
I don't understand your concern. When I said "run it twice", I meant something like
```
java -version 2>&1
java -version >version.txt 2>&1
```
If the first fails, we'll see the output. If the first succeeds, the second is likely to as well and the output should be the same. Yes, if the second fails we won't see its output, but I expect that to be an exceedingly rare situation.

Reply:
> Yes, if the second fails we won't see its output

This was my concern.

> but I expect that to be an exceedingly rare situation.

I do too, but that almost makes it worse. I think it's likely enough to happen at some point, and it will be extremely confusing

Reply:
I don't think we should make failures confusing just because they're rare.

Reply:
The second run could continue to pipe its output through `$(TEE)`. We would get two copies of the output, but I could live with that.

Reply:
That would be the same as the current command, i.e. it could crash without causing `make` to fail

Reply:
Ok, to be robust, perhaps we need to take your approach. Please replace `cat` with `$(CAT)`.